File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.java
class Solution {
  public int titleToNumber(String columnTitle) {
    List<Integer> codes = new ArrayList<Integer>();
    for (int i = 0; i < columnTitle.length(); i++) {
      codes.add((int) columnTitle.charAt(i) - (int) 'A' + 1);
    }
    int n = 0;
    int ans = 0;
    int popped;
    while (!codes.isEmpty()) {
      popped = codes.remove(codes.size() - 1);
      ans += popped * (int) Math.pow(26, n++);
    }
    return ans;
  }
}

File: 0171-excel-sheet-column-number/0171-excel-sheet-column-number.js
/**
 * @param {string} columnTitle
 * @return {number}
 */
var titleToNumber = function(columnTitle) {
  const codes = [];
  for (let i = 0; i < columnTitle.length; i++) {
    codes.push(columnTitle.charCodeAt(i) - 'A'.charCodeAt(0) + 1);
  }
  let [n, ans] = [0, 0];
  let popped;
  while (codes.length) {
    popped = codes.pop();
    ans += popped * 26 ** n++;
  }
  return ans;
};